src/dotnet-compile-fsc/dotnet-compile-fsc.fsproj

```diff
@@ -0,0 +1,62 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
until `preview3` is released, is better to use `preview2` for `src`, because otherwise we can get stuck by a breaking change, without a possibility to update the package easy
so or `preview2` f#, or `preview3` c#